### DISCUSSION TITLE
How can i use devtools in build for test server

### DISCUSSION THREAD

**Discussion by Il9** [jotaijs/jotai-devtools#116] [Q&A]
I'm running development, staging servers for testing
I would like to use devtools in the builds deployed on these servers as well
Is there any way to do this?

**Comment by arjunvegda** [comment]
Hey @Il9 

DevTools is only available when `process.env.NODE_ENV !== 'production'` so it would not work on your deployed environments. 

You could deploy the dev build to your staging servers, however, I'm not sure of the repercussions of it. 🤔

**Reply by Il9** [reply]
I don't understand why the package is exporting the devtools component and disabling it on its own.

**Reply by Il9** [reply]
@arjunvegda 
👀

**Reply by arjunvegda** [reply]
Hey sorry for the delayed response

This is to improve the performance for production builds and to keep your application's bundle sizes low. Note that [this is a limitation in Jotai core library](https://github.com/pmndrs/jotai/blob/main/src/vanilla/store.ts#L745) that jotai-devtools respects.

**Reply by dai-shi** [reply]
Yeah, enabling it means it leaks memory in production.
